### DISCUSSION TITLE
Can this be implemented in Laravel 8?

### DISCUSSION THREAD

**Discussion by jannysurmieda** [dbfx/laravel-strapi#4] [Q&A] [answered]
I'm trying to implement this package in laravel 8 but I always encounter this error: 

Error
Class 'Dbfx\LaravelStrapi' not found
http://localhost:8080/

Can you advised on how to resolve this?

Thank you.

**Comment by dbfx** [comment]
Hi there - it does actually support Laravel 8 by default.  It seems like you might not be including it correctly, can you include a bigger snippet of the code?

Also did your composer install definitely work?

composer require dbfx/laravel-strapi

**Reply by jannysurmieda** [reply]
Thank you for the quick reply. 

The package was successfully installed via composer.  (please see image) 
![Screen Shot 2021-09-03 at 3 55 15 PM](https://user-images.githubusercontent.com/25314087/131970877-c7e46c5d-2d52-494c-a488-5c0c8c561f7f.png)

Here is the bigger snippet of the code

##############

namespace App\Http\Controllers;

use Illuminate\Http\Request;
use Dbfx\LaravelStrapi;

class WelcomeController extends Controller
{
    /**
     * Display a listing of the resource.
     *
     * @return \Illuminate\Http\Response
     */
    public function index()
    {
        //$strapi = new LaravelStrapi();
        $posts = LaravelStrapi::collection('posts');
        dd($posts);
    }

**Comment by dbfx** [comment, marked as answer]
Ah okay great thank you I realise the docs are incorrect. 

```
use Dbfx\LaravelStrapi\LaravelStrapi;

$strapi = new LaravelStrapi();
$posts = $strapi->collection('posts');

```

Can you try like that please?

**Reply by jannysurmieda** [reply]
Thank you. This is the solution to my problem. Thank you very much for the effort.

**Reply by mgaratem** [reply]
This solved my problem too with Laravel 8, pls update the docs for next time ;)

**Reply by dbfx** [reply]
@mgaratem I did update the docs a while ago, are there still old references with the issue?
